plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/controller/FeatureDevController.kt

```diff
@@ -72,8 +71,8 @@ class FeatureDevController(
     private val authController: AuthController = AuthController()
 ) : InboundAppMessagesHandler {
 
-    private val messenger = context.messagesFromAppToUi
```

Comment:
Can we not use `.replaceService` in test instead of making this public?

Reply:
I'll try

Reply:
I need these variables public so I can have the extensions